### DISCUSSION TITLE
diskusi

### DISCUSSION THREAD

**Discussion by bangroy20** [bangroy20/tokokita#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kedawon4636** [comment]
bagus

**Reply by bangroy20** [reply]
yawis

**Comment by ISmylife13** [comment]
**gasss**

**Reply by bangroy20** [reply]
yawis

**Comment by ISmylife13** [comment]
<div class="container">
        <div class="card p-3" style="max-width: 350px; border-radius:20px;">

          <div class="card-body" style="background-color:#FFE400; border-radius:15px;">
            <h4 class="card-title">YOU SHOP </h4><br>
            <div>
              <img src="../YOU SHOP/file/login.jpg" width="100px" class="rounded-circle">
            </div>
            <h4 class="card-title">Login</h4><br>
